dcos_installer/action_lib.py

```diff
@@ -353,18 +353,48 @@ def uninstall_dcos(config, block=False, state_json_dir=None, async_delegate=None
 def _add_prereqs_script(chain):
     inline_script = """
 #/bin/sh
```

Comment:
It should be `#!/bin/sh` - SHEBANG.

Reply:
agreed let me fix this.

Reply:
As per our discussion won't make this change since currently everything was probably being run through `bash` and this change would end up forcing the scripts to use `sh` which might break things.